app/views/needs/_met_when.html.erb

```diff
@@ -1,9 +1,5 @@
 
-<%= f.action  :submit,
-              :button_html => {
-                :value => "Save",
-                :class => "btn btn-default btn-primary hide"
-              } %>
+<%= f.submit "Save", class: "btn btn-default btn-primary hide"%>
```

Comment:
nitpick lol sorry but should be a space between `hide"` and the closing tag `%>`